localstack/services/dynamodb/dynamodb_starter.py

```diff
@@ -10,7 +10,7 @@
 LOGGER = logging.getLogger(__name__)
 
 # max heap size allocated for the Java process
-MAX_HEAP_SIZE = '256m'
+MAX_HEAP_SIZE = config.DYNAMODB_HEAP_SIZE
```

Comment:
Better remove the constant from here, and use `config.DYNAMODB_HEAP_SIZE` directly in the `start_dynamodb(..)` function. (This will allow for lazy initialization and overwriting of the configuration variable at runtime.)

Reply:
I don't know why I didn't think of that.. I was just following the code already in the file